### PR TITLE
update EXO report docs

### PR DESCRIPTION
**Pull request by sarahwxy**
Migrated the Angular reports to React. Updating docs to reflect the changes.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+search.appverid:
+- BCS160
+- MET150
+- MOE150
+- GEA150
+ms.assetid: 1cbe2c00-ca65-4fb9-9663-1bbfa58ebe44
+description: "Learn how to get an email activity report using the Microsoft 365 Reports dashboard in the Microsoft 365 admin center."
+---
+
+# Microsoft 365 Reports in the admin center - Email activity
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:24]:
```suggestion
# Microsoft 365 Reports in the admin center - email activity
```

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The Microsoft 365 **Reports** dashboard shows you the activity overview across the products in your organization. It enables you to drill in to individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:26]:
```suggestion
The Microsoft 365 **Reports** dashboard shows you the activity overview across the products in your organization. It enables you to drill into individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md).
```

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  
+For example, you can get a high level view of email traffic within your organization from the Reports page, and then you can drill into the Email activity widget to understand the trends and per user level details of the email activity within your organization.
+  
+> [!NOTE]
+> You must be a global administrator, global reader or reports reader in Microsoft 365 or an Exchange, SharePoint, Teams Service, Teams Communications, or Skype for Business administrator to see reports.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:31]:
```suggestion
> You must be a global administrator, global reader, or reports reader in Microsoft 365 or an Exchange, SharePoint, Teams Service, Teams Communications, or Skype for Business administrator to see reports. 
```

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+You can get a view into your user's email activity by looking at the **Activity** and **Users** charts. 
+  
+![Email activity report](../../media/5eb1d9e9-8106-4843-acb7-c0238c0da816.png)
+  
+|Item|Description|
+|:-----|:-----|
+|1.  <br/> |The **Email activity** report can be viewed for trends over the last 7 days, 30 days, 90 days, or 180 days. However, if you select a particular day in the report, the table (7) will show data for up to 28 days from the current date (not the date the report was generated).  <br/> |
+|2.  <br/> |The data in each report usually covers up to the last 24 to 48 hours.  <br/> |
+|3.  <br/> |The **Activity** chart enables you to understand the trend of the amount of email activity going on in your organization. You can understand the split of email send, email read, email received, meeting created, or meeting interacted activities.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:49]:
```suggestion
|3.  <br/> |The **Activity** chart enables you to understand the trend of the amount of email activity going on in your organization. You can understand the split of email send, email read, email received, meetings created, or meeting interacting activities.  <br/> |
```

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+|4.  <br/> |The **User** chart enables you to understand the trend of the amount of unique users who are generating the email activities. You can look at the trend of users performing email sending, email reading, email receiving, meeting creating, or meeting interacting activities.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:50]:
```suggestion
|4.  <br/> |The **User** chart enables you to understand the trend of the number of unique users who are generating the email activities. You can look at the trend of users performing email sending, email reading, email receiving, meeting creating, or meeting interacting activities.  <br/> |
```

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+|5.  <br/> | On the **Activity** chart, the Y axis is the count of activity of the type email sent, email received, email read, meeting created, and meeting interacted.  <br/>  On the **Users** activity chart, the Y axis is the user's performing activity of the type email sent, email received, email read, meeting created, or meeting interacted.  <br/>  The X axis on both charts is the selected date range for this specific report.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:51]:
```suggestion
|5.  <br/> | On the **Activity** chart, the Y-axis is the count of activity of the type email sent, email received, email read, meetings created, and meetings interacted.  <br/>  On the **Users** activity chart, the Y-axis is the user's performing activity of the type email sent, email received, email read, meetings created, or meetings interacted.  <br/>  The X-axis on both charts is the selected date range for this specific report.  <br/> |
```

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+|8.  <br/> |Select **Choose columns** to add or remove columns from the report.  <br/> ![Email activity report - choose columns](../../media/80ffa0ad-61c5-4a6f-8a1d-5f6730ff7da9.png)|
+|9.  <br/> |You can also export the report data into an Excel .csv file, by selecting the **Export** link. This exports data of all users and enables you to do simple sorting and filtering for further analysis. If you have less than 2000 users, you can sort and filter within the table in the report itself. If you have more than 2000 users, in order to filter and sort, you will need to export the data.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:55]:
```suggestion
|9.  <br/> |You can also export the report data into an Excel .csv file, by selecting the **Export** link. This exports data of all users and enables you to do simple sorting and filtering for further analysis. If you have less than 2,000 users, you can sort and filter within the table in the report itself. If you have more than 2,000 users, in order to filter and sort, you will need to export the data.  <br/> |
```

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+|||
+   
+Note: The Email activity report is only available for mailboxes that are associated with users who have licenses.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:58]:
```suggestion
> [!NOTE]
> The Email activity report is only available for mailboxes that are associated with users who have licenses.
```

File: microsoft-365/admin/activity-reports/email-apps-usage-ww.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+- Adm_O365
+- Adm_NonTOC
+ms.custom: AdminSurgePortfolio
+search.appverid:
+- BCS160
+- MET150
+- MOE150
+- GEA150
+ms.assetid: c2ce12a2-934f-4dd4-ba65-49b02be4703d
+description: "Learn how to get Email apps usage report to know about email apps connecting to Exchange Online and the Outlook version users are using."
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage-ww.md:20]:
```suggestion
description: "Learn how to get email apps usage reports to know about email apps connecting to Exchange Online and the Outlook version users are using."
```

File: microsoft-365/admin/activity-reports/email-apps-usage-ww.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+# Microsoft 365 Reports in the admin center - Email apps usage
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage-ww.md:23]:
```suggestion
# Microsoft 365 Reports in the admin center - email apps usage
```

File: microsoft-365/admin/activity-reports/email-apps-usage-ww.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+The Microsoft 365 **Reports** dashboard shows you the activity overview across the products in your organization. It enables you to drill in to individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the email apps usage report, you can see how many email apps are connecting to Exchange Online. You can also see the version information of Outlook apps that users are using, which will allow you to follow up with those who are using unsupported versions to install supported versions of Outlook.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage-ww.md:25]:
```suggestion
The Microsoft 365 **Reports** dashboard shows you the activity overview across the products in your organization. It enables you to drill into individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the email apps usage report, you can see how many email apps are connecting to Exchange Online. You can also see the version information of Outlook apps that users are using, which will allow you to follow up with those who are using unsupported versions to install supported versions of Outlook.
```

File: microsoft-365/admin/activity-reports/email-apps-usage-ww.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+  
+> [!NOTE]
+> You must be a global administrator, global reader or reports reader in Microsoft 365 or an Exchange, SharePoint, Teams Service, Teams Communications, or Skype for Business administrator to see reports.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage-ww.md:28]:
```suggestion
> You must be a global administrator, global reader, or reports reader in Microsoft 365 or an Exchange, SharePoint, Teams Service, Teams Communications, or Skype for Business administrator to see reports.  
```

File: microsoft-365/admin/activity-reports/email-apps-usage-ww.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+|7.  <br/> |You can filter the series you see on the chart by selecting an item in the legend.  <br/> |
+|8.  <br/> | You might not see all the items in the list below in the columns until you add them.<br/> **Username** is the name of the email app's owner.  <br/> **Last activity date** is the latest date the user read or sent an email message.  <br/> **Mac mail**, **Mac Outlook** and **Outlook**, **Outlook mobile** and **Outlook on the web** are examples of email apps you may have in your organization.  <br/>  If your organization's policies prevents you from viewing reports where user information is identifiable, you can change the privacy setting for all these reports. Check out the **How do I hide user level details?** section in the [Activity Reports in the Microsoft 365 admin center](activity-reports.md).  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage-ww.md:51]:
```suggestion
|8.  <br/> | You might not see all the items in the list below in the columns until you add them.<br/> **Username** is the name of the email app's owner.  <br/> **Last activity date** is the latest date the user read or sent an email message.  <br/> **Mac mail**, **Mac Outlook** and **Outlook**, **Outlook mobile**, and **Outlook on the web** are examples of email apps you may have in your organization.  <br/>  If your organization's policies prevent you from viewing reports where user information is identifiable, you can change the privacy setting for all these reports. Check out the **How do I hide user level details?** section in the [Activity Reports in the Microsoft 365 admin center](activity-reports.md).  <br/> |
```

File: microsoft-365/admin/activity-reports/email-apps-usage-ww.md [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+|9.  <br/> |Select **Choose columns** to add or remove columns from the report.  <br/> ![Email apps usage report - choose columns](../../media/041bd6ff-27e8-409d-9608-282edcfa2316.png)|
+|10.  <br/> |You can also export the report data into an Excel .csv file, by selecting the **Export** link. This exports data of all users and enables you to do simple sorting and filtering for further analysis. If you have less than 2000 users, you can sort and filter within the table in the report itself. If you have more than 2000 users, in order to filter and sort, you will need to export the data.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage-ww.md:53]:
```suggestion
|10.  <br/> |You can also export the report data into an Excel .csv file, by selecting the **Export** link. This exports data of all users and enables you to do simple sorting and filtering for further analysis. If you have less than 2,000 users, you can sort and filter within the table in the report itself. If you have more than 2,000 users, in order to filter and sort, you will need to export the data.  <br/> |
```

File: microsoft-365/admin/activity-reports/email-apps-usage-ww.md [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+|3.  <br/> |The **Users** view shows you the number of unique users that connected to Exchange Online using any email app.  <br/> |
+|4.  <br/> |The **Apps** view shows you the number of unique users by app over the selected time period.  <br/> |
+|5.  <br/> |The **Versions** view shows you the number of unique users for each version of Outlook in Windows.  <br/> |
+|6.  <br/> | On the **Users** chart, the Y axis is the total count of unique users that connected to an app on any day of the reporting period.  <br/>  On the **Users** chart, the X axis is number of unique users that used the app for that reporting period.  <br/>  On the **Apps** chart, the Y axis is the total count of unique users who used a specific app during the reporting period.  <br/>  On the **Apps** chart, the X axis is the list of apps in your organization.  <br/>  On the **Versions** chart, the Y axis is the total count of unique users using a specific version of Outlook desktop. If the report can't resolve the version number of Outlook, the quantity will show as **Undetermined**.  <br/>  On the **Versions** chart, the X axis is the list of apps in your organization.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage-ww.md:49]:
```suggestion
|6.  <br/> | On the **Users** chart, the Y-axis is the total count of unique users that connected to an app on any day of the reporting period.  <br/>  On the **Users** chart, the X-axis is the number of unique users that used the app for that reporting period.  <br/>  On the **Apps** chart, the Y-axis is the total count of unique users who used a specific app during the reporting period.  <br/>  On the **Apps** chart, the X-axis is the list of apps in your organization.  <br/>  On the **Versions** chart, the Y-axis is the total count of unique users using a specific version of Outlook desktop. If the report can't resolve the version number of Outlook, the quantity will show as **Undetermined**.  <br/>  On the **Versions** chart, the X-axis is the list of apps in your organization.  <br/> |
```

File: microsoft-365/admin/activity-reports/email-activity-ww.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
...s deleted, but was active during some part of the reporting period of the report.  <br/> **Deleted date** is the date the user was deleted.  <br/> **Last activity date** refers to the last time the user performed a read or send email activity.  <br/> **Send actions** is the number of times an email send action was recorded for the user.  <br/> **Receive actions** is the number of times an email received action was recorded for the user.  <br/> **Read actions** is the number of times an email read action was recorded for the user.  <br/> **Meeting created actions** is the number of times a meeting request send action was recorded for the user.  <br/> **Meeting interacted actions** is the number of times a meeting request accept, tentative, decline, or cancel action was recorded for the user.  <br/> **Product assigned** is the products that are assigned to this user.  <br/>  If your organization's policies prevents you from viewing reports where user information is identifiable, you can change the privacy setting for all these reports. Check out the **How do I hide user level details?** section in the [Activity Reports in the Microsoft 365 admin center](activity-reports.md).  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-activity-ww.md:53]:
```suggestion
|7.  <br/> | The table shows you a breakdown of the email activities at the per-user level. This shows all users that have an Exchange product assigned to them and their email activities. <br/> <br/> **Username** is the email address of the user.  <br/> **Display name** is the full name if the user.  <br/> **Deleted** refers to the user whose current state is deleted, but was active during some part of the reporting period of the report.  <br/> **Deleted date** is the date the user was deleted.  <br/> **Last activity date** refers to the last time the user performed a read or send email activity.  <br/> **Send actions** is the number of times an email send action was recorded for the user.  <br/> **Receive actions** is the number of times an email received action was recorded for the user.  <br/> **Read actions** is the number of times an email read action was recorded for the user.  <br/> **Meeting created actions** is the number of times a meeting request send action was recorded for the user.  <br/> **Meeting interacted actions** is the number of times a meeting request accept, tentative, decline, or cancel action was recorded for the user.  <br/> **Product assigned** is the products that are assigned to this user.  <br/>  If your organization's policies prevent you from viewing reports where user information is identifiable, you can change the privacy setting for all these reports. Check out the **How do I hide user level details?** section in the [Activity Reports in the Microsoft 365 admin center](activity-reports.md).  <br/> |
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/activity-reports/email-activity-ww.md:53]: Please change " **Display Name** is the full name if the user." to " **Display Name** is the full name of the user." in the fourth line and "If your organization's policies prevents you..." to "If your organization's policies prevent you..." in the second last line.

File: microsoft-365/admin/activity-reports/mailbox-usage.md [hunk 17]
```diff
@@ -49,6 +50,7 @@ You can get a view into your organization's **Mailbox usage** by looking at the
 |6.  <br/> | On the **Mailbox** chart, the Y axis is the count of user mailboxes.  <br/>  On the **Storage** chart, the Y axis is the amount of storage being used by user mailboxes in your organization.  <br/>  On the **Quota** chart, the Y axis is the number of user mailboxes in each storage quota.  <br/>  The X axis on the Mailbox and Storage charts is the selected date range for this specific report.  <br/>  The X axis on the Quota charts is the quota category.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/mailbox-usage.md:50]:
```suggestion
|6.  <br/> | On the **Mailbox** chart, the Y-axis is the count of user mailboxes.  <br/>  On the **Storage** chart, the Y-axis is the amount of storage being used by user mailboxes in your organization.  <br/>  On the **Quota** chart, the Y-axis is the number of user mailboxes in each storage quota.  <br/>  The X-axis on the Mailbox and Storage charts is the selected date range for this specific report.  <br/>  The X-axis on the Quota charts is the quota category.  <br/> |
```

File: microsoft-365/admin/activity-reports/mailbox-usage.md [hunk 18]
```diff
@@ -49,6 +50,7 @@ You can get a view into your organization's **Mailbox usage** by looking at the
...
... was deleted.  <br/> **Create date** is the date the mailbox was created.  <br/> **Last activity date** refers to the date the mailbox had an email send or read activity.  <br/> **Item count** refers to the total number of items in the mailbox.  <br/> **Storage used (MB)** refers to the total storage used.  <br/> **Deleted Item Count** refers to the total number of deleted items in the mailbox. <br/> **Deleted Item Size (MB)** refers to the total size of all deleted items in the mailbox. <br/> **Issue warning quota (MB)** refers to the storage limit when the mailbox owner will receive a warning that it's about to hit the storage quota.  <br/> **Prohibit send quota (MB)** refers to the storage limit when the mailbox can no longer send emails.  <br/> **Prohibit send receive quota (MB)** refers to the storage limit when the mailbox can no longer send or receive emails.  <br/>  If your organization's policies prevents you from viewing reports where user information is identifiable, you can change the privacy setting for all these reports. Check out the **Hide user details in the reports** section in the [Activity Reports in the Microsoft 365 admin center](activity-reports.md).  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/mailbox-usage.md:52]:
```suggestion
|8.  <br/> | The table shows you a breakdown of mailbox usage at the per-user level. You can add additional columns to the table.  <br/> **User name** is the email address of the user.  <br/> **Display Name** is the full name of the user.  <br/> **Deleted** refers to the mailbox whose current state is deleted, but was active during some part of the reporting period of the report.  <br/> **Deleted date** is the date the mailbox was deleted.  <br/> **Create date** is the date the mailbox was created.  <br/> **Last activity date** refers to the date the mailbox had an email send or read activity.  <br/> **Item count** refers to the total number of items in the mailbox.  <br/> **Storage used (MB)** refers to the total storage used.  <br/> **Deleted Item Count** refers to the total number of deleted items in the mailbox. <br/> **Deleted Item Size (MB)** refers to the total size of all deleted items in the mailbox. <br/> **Issue warning quota (MB)** refers to the storage limit when the mailbox owner will receive a warning that it's about to hit the storage quota.  <br/> **Prohibit send quota (MB)** refers to the storage limit when the mailbox can no longer send emails.  <br/> **Prohibit send receive quota (MB)** refers to the storage limit when the mailbox can no longer send or receive emails.  <br/>  If your organization's policies prevent you from viewing reports where user information is identifiable, you can change the privacy setting for all these reports. Check out the **Hide user details in the reports** section in the [Activity Reports in the Microsoft 365 admin center](activity-reports.md).  <br/> |
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/activity-reports/mailbox-usage.md:52]: Please change " **Display Name** is the full name if the user." to " **Display Name** is the full name of the user." in the fourth line and "If your organization's policies prevents you..." to "If your organization's policies prevent you..." in the second last line.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**